Content.Server/Electrocution/ElectrocutionSystem.cs

```diff
@@ -266,7 +267,7 @@ private void OnElectrifiedInteractUsing(EntityUid uid, ElectrifiedComponent elec
         {
             if (id != null &&
                 nodeContainer.TryGetNode<Node>(id, out var tryNode) &&
-                tryNode.NodeGroup is IBasePowerNet { NetworkNode: { LastCombinedSupply: > 0 } })
+                (tryNode.NodeGroupID == NodeGroupID.AMEngine || tryNode.NodeGroup is IBasePowerNet { NetworkNode: { LastCombinedSupply: > 0 } }))
```

Comment:
Nodegroups shouldn't be hardcoded in here.

Reply:
Check here: https://github.com/space-wizards/space-station-14/pull/16963#issuecomment-1571742420

Reply:
I wasn't aware of such component, I'll see if I can make use of it o.O